packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -262,6 +262,12 @@ const TRANSFORMS = {
         capability.startsWith('hvm')
       ),
 
+      // Only exists on XCP-ng/XenServer >= 8.2
+      certificates: obj.$certificates?.map(({ fingerprint, not_after }) => ({
+        fingerprint,
+        expiry: toTimestamp(not_after),
```

Comment:
`expiry` does not appear to be common, maybe something else would be better: *expirationDate*, *expires*, *notAfter*?

Reply:
I used `expiry` because it's used in the same UI for licenses. IMHO, we shouldn't use multiple names with the same signification in the same interface. 

See https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/host/tab-advanced.js#L419

Reply:
Ah OK, do you mean to not use this term in the object properties, but let it in the display. I'm ok with notAfter.